src/resolvers/Query/advertisementsConnection.ts

```diff
@@ -0,0 +1,22 @@
+import type { QueryResolvers } from "../../types/generatedGraphQLTypes";
```

Comment:
@disha1202  Just the name of the file was changed, the internal logic wasn't changed to be a connection.